lib/src/language.h

```diff
@@ -10,6 +10,9 @@ extern "C" {
 
 #define ts_builtin_sym_error_repeat (ts_builtin_sym_error - 1)
 
+#define LANGUAGE_VERSION_WITH_PRIMARY_STATES 14
+#define LANGUAGE_VERSION_USABLE_VIA_WASM 13
```

Comment:
Having a `NOTE`/`TODO` to drop this when we get to abi 15/1.0 might be useful to include (easy to find)

Reply:
I think it would be ok to keep these around even if we change the API, unless we do so in a backward-incompatible change (and bump the MIN_COMPATIBLE_VERSION).